tests/unit/kernels_reference.cpp

```diff
@@ -104,6 +104,20 @@ void dw::reference::CPUReLUBackward(const float* in, const float* grad_output,
                    [](float in, float go) { return in > 0.0 ? go : 0.0;});
 }
 
+void CPUELUForward(const float* in, float* out, float alpha, size_t size) {
```

Comment:
Stop using raw pointers in reference signature, because it isn't descriptive

Reply:
Done.